mesheryctl/internal/cli/root/system/check.go

```diff
@@ -254,6 +254,10 @@ func (hc *HealthChecker) runDockerHealthChecks() error {
 	if hc.Options.PrintLogs {
 		log.Info("\nDocker \n--------------")
 	}
+	endpointParts := strings.Split(hc.context.Endpoint, ":")
+	if !strings.HasSuffix(endpointParts[1], "localhost") {
```

Comment:
Why not using `!=` instead of `!strings.HasSuffix`

Reply:
Are you suggesting using `!= "//localhost"`?. I chose `strings.HasSuffix` because it was already used in other parts of the code. Can you please explain your reasoning behind the suggestion?

Reply:
> Are you suggesting using != "//localhost"?

Yes

> Can you please explain your reasoning behind the suggestion?

You don't really want to check the suffix i.e, if `//notlocalhost` was sent it would pass. Although you may see the example that I gave as an absurd one but my point is we're not trying to check the suffix truly, we're checking if it is `//localhost`.

Reply:
Got it. I'll make the changes. Thanks for the clarification